airmozilla/manage/views.py

```diff
@@ -3687,6 +3688,18 @@ def picture_delete(request, id):
 
 
 @staff_required
+@permission_required('main.delete_picture')
+@transaction.commit_on_success
+@json_view
+def picture_delete_all(request):
+    event = get_object_or_404(Event, id=request.POST.get('event'))
```

Comment:
Can you change this to:
```python
event = get_object_or_404(Event, id=request.POST['event'])
```
The reason is that you only use `.get()` if it's "ok" to not be there. It then automatically becomes `None` so you could potentially do: `event = get_object_or_404(Event, id=None)`. It would fail but it would be cryptic why or how it failed.